indico/modules/users/templates/users_admin.html

```diff
@@ -14,9 +14,20 @@
                     <div class="group">
                         <a class="i-button icon-users" href="{{ url_for('.users_merge') }}">Merge users</a>
                     </div>
+                    <div class="group">
+                        <a class="i-button icon-plus" href="{{ url_for('.users_create') }}">Create a new account</a>
+                    </div>
                 </div>
             </div>
             <div class="i-box-content">
+                <h2>{% trans %} Account creation settings {% endtrans %} </h2>
+                {{ form_header(user_management_form) }}
+                {{ form_rows(user_management_form) }}
+                {% call form_footer(user_management_form) %}
+                    <input class="i-button big highlight" type="submit" value="{% trans %}Save{% endtrans %}" name="save_management_form">
```

Comment:
`name="save_management_form"` - you could use a hidden input instead or even use a `SubmitButton` in the form definition itself (skip it in the `form_rows()` call and render it using `form.whatever(class_='i-button big highlight')`).